pkg/scripts/cleanup.sh

```diff
@@ -7,11 +7,13 @@
 # -h: help
 set -e
 CURRENT_PATH="$(pwd)"
```

Comment:
Do we still use `CURRENT_PATH` somewhere?

Reply:
yes, in line 13

Here we get current path, change folder to root path and then back to current path. The reason is to get the clean root path of project instead of having the script path follow by `../../`